modules/apps/frontend-js/frontend-js-translation-web/src/main/resources/META-INF/resources/TranslationSelector.tsx

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayForm from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayLayout from '@clayui/layout';
+import React from 'react';
+
+interface IItem {
+	label: string;
+	symbol: string;
+}
+
+interface ITranslations {
+	[key: string]: string;
+}
+
+export type ButtonSize = null | 'regular' | 'small';
+
+export type Variant =
+	| null
+	| 'default'
+	| 'language selector'
+	| 'translation selector';
```

Comment:
Perhaps in this case it would be better to define shorter names for variation, for example: `type Variant = 'language' | 'translation';`, the `default` can always be when the property value is `undefined`. What do you think?

Reply:
I think it's a good proposal, going to update

Reply:
Updated in [`09a16fd`](https://github.com/liferay-frontend/liferay-portal/pull/1184/commits/09a16fd28de164ce4eb3ebe6162531ca0c30cd91)